apps/main/src/loan/entities/scrvusdStatistics.ts

```diff
@@ -30,7 +31,7 @@ const VAULT_ABI = [
  * If a provider is provided, the data is fetched from the vault contract directly.
  * That provides more accurate data and works even if our servers are down.
  */
-async function _fetchSavingsStatistics(): Promise<PricesStatisticsDataResponse> {
+async function _fetchSavingsStatistics(): Promise<Statistics> {
```

Comment:
I think the return type is implicit
```suggestion
async function _fetchSavingsStatistics() {
```